python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -464,8 +474,8 @@ async def latiss_acquire(self):
                         z=-self.manual_focus_offset
                     )
                     self.manual_focus_offset_applied = False
-                self.log.exception(e)
-                raise e
+                self.log.exception("Centroid finding algorithm was unsuccessful.")
+                raise RuntimeError("Centroid finding algorithm was unsuccessful.")
```

Comment:
If you are raising the exception here, you probably don't need to the log message above.